yt/visualization/volume_rendering/render_source.py

```diff
@@ -901,7 +901,7 @@ def annotate_mesh_lines(self, color=None, alpha=1.0):
         if color is None:
             color = np.array([0, 0, 0, alpha])
 
-        locs = [self.sampler.amesh_lines == 1]
+        locs = tuple(self.sampler.amesh_lines == 1)
```

Comment:
Are there any times this could result in an expensive array-to-tuple conversion? Would it be better to do `(self.sampler..., )`?

Reply:
good point, indeed the conversion is expensive for large arrays.
I now think the best approach would simply be
```python
     locs = self.sampler.amesh_lines == 1
```

Reply:
Welp I won't investigate why this broke the the docs build. I'm switching to your suggestion